images/win/scripts/Installers/Install-Rust.ps1

```diff
@@ -11,6 +12,19 @@ $env:CARGO_HOME = "C:\Users\Default\.cargo"
 # See https://rustup.rs/#
 $rustupPath = Start-DownloadWithRetry -Url "https://static.rust-lang.org/rustup/dist/x86_64-pc-windows-msvc/rustup-init.exe" -Name "rustup-init.exe"
 
+#region Supply chain security
+Write-Verbose "Performing checksum verification"
+$local_file_hash = (Get-FileHash -Path (Join-Path ${env:TEMP} 'rustup-init.exe') -Algorithm SHA256).Hash
```

Comment:
Still thinking about camelCase for the variables names.

Reply:
thank you for the suggestion, much better indeed